_posts/en/posts/2016-02-26-zkcp.md

```diff
@@ -0,0 +1,140 @@
+---
+type: posts
+layout: post
+lang: en
+id: en-zkcp-announce
+name: zkcp-announce
+title: The first successful Zero-Knowledge Contingent Payment
+permalink: /en/2016/02/26/zkcp-announce/
+author: gmaxwell
+tags: [privacy]
+version: 1
+excerpt: Announcement for the first Zero-Knowledge Contingent Payment on the Bitcoin Network.
+---
+I am happy to announce the first successful  Zero-Knowledge Contingent Payment (ZKCP) on the Bitcoin Network.
+
+ZKCP is a transaction protocol that allows a buyer to purchase information from a seller using Bitcoin in a manner which is private, scalable, secure, and which doesn't require trusting anyone: the expected information is transferred if and _only_ if the payment is made. The buyer and seller do not need to trust each other or depend on arbitration by a third party.
```

Comment:
We should probably mention later in the article, either today or at another time, that there are certain security assumptions required, such as that hash functions work or something. I guess this goes without saying most of the time. Certainly these disclaimers don't have to be made in this particular paragraph.